docs/reference/mapping/types/geo-shape.asciidoc

```diff
@@ -127,6 +127,20 @@ the `tree` or `strategy` parameters according to the appropriate
 <<geo-shape-mapping-options>>. Note that these parameters are now deprecated
 and will be removed in a future version.
 
+*IMPORTANT NOTES*
+
+The following features are not yet supported with the new indexing approach:
+
+* `geo_shape` query with `MultiPoint` geometry types - Elasticsearch currently prevents searching
```

Comment:
Do we actually raise an error if a user uses a multi-point to search, or do we return wrong results?

Reply:
We issue a `QueryShardException` stating `MultiPoint` queries are not supported.

Reply:
Great, thanks for confirming.